tests/pr_test.go

```diff
@@ -18,7 +18,6 @@ const ocpVersion1 = "4.12"
 const ocpVersion2 = "4.11"
 const ocpVersion3 = "4.10"
 const ocpVersion4 = "4.9"
```

Comment:
Why do we not have other_test.go file like we had internally? We need to ensure we have a test for all of these versions.

Reply:
Noted. Will add other_tests.go